### ISSUE TITLE
minetest.send_chat_message() désactivé par défaut

### ISSUE THREAD

**Issue by mgl512** [MT-Eurythmia/Eurythmia-server#119]
J'ai remarqué ça dans mon dernier pull du dépôt Minetest. minetest.send_chat_message() est désactivé par défaut par ce commit: [[CSM] Add flavour limits controlled by server (#5930)](https://github.com/minetest/minetest/commit/79f19b8369a0120fbf0bd0ee3739989fbbbca50e)

C'est désactivé par le client, je ne peux plus envoyer mes textes colorés. Je n'ai pas examiné le commit mais je suppose qu'il faut écrire un mod server pour réactiver cette possibilité.

**Comment by paly2** [comment]
Non, c'est un réglage dans le `minetest.conf`. Ceci dit il n'est peut-être pas plus mal de conserver cette fonctionnalité désactivée. @mgl512, @matt8fr, @mega-gigaMynetest, @fwhcat ? (Mon propre avis sur la question est assez neutre.)

**Comment by mgl512** [comment]
> Mon propre avis sur la question est assez neutre

Mais elle vient pas de toi la pull request minetest_send_chatmessage() ? Tu ne vas quand même pas la désactiver sur ton propre serveur ?  En plus ça permet d'envoyer plein de caractères de controle sur l'IRC. Pour une fois que c'est dans ce sens-là que c'est illisible...

**Comment by paly2** [comment]
La PR est bien de moi, mais j'imagine que tout le monde n'aime pas beaucoup le fait que le client puisse envoyer des messages qu'il n'a pas tapés. Personnellement je trouve que ça ne changerait pas grand-chose actuellement sur le serveur (il y aurait juste coloured_chat qui ne fonctionnerait plus ; notons qu'il existe un autre réglage dans le minetest.conf pour désactiver spécifiquement le chat coloré en provenance du client et qui prend effet côté serveur), et que de toute façon il est très facile de hacker le client pour faire disparaître la prise en compte de ce réglage (même pas besoin de le hacker d'ailleurs, il suffit d'utiliser 0.4.16-stable).

Je réactive pour le moment, mais je suis toujours à l'écoute des avis.

**Comment by mega-gigaMynetest** [comment]
Le fait de l'auto-chat ne me derange pas du tout, mais je n'aime pas les messages en couleurs, ça met en valeur n'importe qui plus que les autres, moi je les désactiverais.

**Comment by matt8fr** [comment]
j’écris principalement en blanc maintenant mais j’apprécie de pouvoir faire passer un message en couleur pour annoncer quelque chose ou recadrer un joueur par exemple, après si tu remplace colored_chat par un mod serveur genre: " /color red <msg> " ça me va aussi.
pour l'IRC je ne sais pas je n'utilise pas.
